code.js

```diff
@@ -1,3 +1,28 @@
 function divideAndConquerSum(a) {
-    return a;
+	//console.log(a);
+	if (a.length == 0)
+		return 0;
+	return (arrayBreakUp(0, a.length-1, a)+ a[a.length-1]);
 }
+
+function arrayBreakUp(lo, hi, tmp) {
+	if(hi - lo <= 1){
+		//console.log(tmp[lo]);
+		//console.log(tmp[hi]);
+		//console.log(" ");
+		if (lo == hi)
+			return 0
+		else if (lo > hi)
+			return tmp[hi]
```

Comment:
Not sure if this makes sense -- if `lo` is higher than `hi`, something must have gone wrong, right?